tests/contrib/executors/test_kubernetes_executor.py

```diff
@@ -459,10 +489,36 @@ def test_make_pod_run_as_user_0(self):
 
         pod = worker_config.make_pod("default", str(uuid.uuid4()), "test_pod_id", "test_dag_id",
                                      "test_task_id", str(datetime.utcnow()), 1, "bash -c 'ls /'",
-                                     kube_executor_config)
+                                     prestop_wait_time=0,
+                                     kube_executor_config=kube_executor_config)
 
         self.assertEqual(0, pod.security_context['runAsUser'])
 
+    def test_make_pod_assert_labels(self):
+        # Tests the pod created has all the expected labels set
+        self.kube_config.dags_folder = 'dags'
+
+        worker_config = WorkerConfiguration(self.kube_config)
+        kube_executor_config = KubernetesExecutorConfig(annotations=[],
+                                                        volumes=[],
+
+                                                        volume_mounts=[])
+        pod = worker_config.make_pod("default", "sample-uuid", "test_pod_id", "test_dag_id",
+                                     "test_task_id", "2019-11-21 11:08:22.920875", 1, "bash -c 'ls /'",
+                                     prestop_wait_time=0,
+                                     kube_executor_config=kube_executor_config)
+        expected_labels = {
+            'airflow-worker': 'sample-uuid',
+            'airflow_version': airflow_version.replace('+', '-'),
+            'dag_id': 'test_dag_id',
+            'execution_date': '2019-11-21 11:08:22.920875',
+            'kubernetes_executor': 'True',
+            'my_label': 'label_id',
+            'task_id': 'test_task_id',
+            'try_number': '1'
```

Comment:
This should pass one of the failing test

```suggestion
        expected_labels = {
            'airflow-worker': 'sample-uuid',
            'dag_id': 'test_dag_id',
            'execution_date': '2019-11-21 11:08:22.920875',
            'my_label': 'label_id',
            'task_id': 'test_task_id',
            'try_number': '1'
```